modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,22 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option allows you to split the current container block element when the enter key is pressed, and specify which blocks should be split.
```

Comment:
A minor edit to @lnewson’s edit plus some presentational changes:
```suggestion
This option specifies how the current container block element is split when the Enter key is pressed inside an empty inner block element. The available options are:

* always split the block;
* split for specific block elements; or
* do not split the block at all.
```
When discussing a lists of options, it’s always worth considering if presenting said options as a literal list helps.

I think it does here. The first sentence above document how and when. And the second sentence sets out what, and lays out the what in a list for clarity’s sake.